tests/test_examples.py

```diff
@@ -55,6 +55,7 @@ def run_main_subproc(args):
             "--no-show-video",
             "--no-make-video",
         ),
+        ("examples/tutorials/nb_python/ECCV_2020_Navigation.py", "--no-show-images"),
```

Comment:
Is the --no-show-images arg even parsed anywhere by the script? If not, you can just drop that arg.